GoodDollar/GoodDAPP
# (Chore) Skip the zoom component during cypress tests runs

# Description

If cypress is running:
-skip preload call
- in useZoomSDK dont call initialize, just return initialized = true
-in useZoomVerification dont create session, just call FaceVerificationAPI.enroll() method with any fake data - it will be successfully processed on the server
- then verificaion screen will receive success callback and we'll be redirected to claim as usual

# How Has This Been Tested?

1. Install any extension customizing user agent string
2. set custom user agent `Mozilla/5.0 (X11; Linux x86_64; Cypress) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36`
3. Create new account, try to claim
4. After FR dialog shown, click "Ok verify me"
5. You should immediately pass verification without capturing your face 

# Checklist:
